### PR TITLE
Ejercicios día 1

### PR DESCRIPTION
**Pull request by ElenaMLopez**
Hope you like it!!!!

🦄 🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄

### CODE REVIEW

**Comment by ElenaMLopez** [comment]
@UlisesGascon take a look please

File: elena/ejercicios1/Twitter.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Ponemos en consola:
+
```
Comment (UlisesGascon) [inline elena/ejercicios1/Twitter.js:2]: Muuuuy elegante!! :-)

**Review by UlisesGascon — APPROVED** [review]
Bravo! :clap::clap:

**Comment by UlisesGascon** [comment]
Ya Tienes permisos!

**Pull request merged**